client/src/main/java/sjt/http/client/module/request/HttpRequest.java

```diff
@@ -0,0 +1,134 @@
+package sjt.http.client.module.request;
+
+import sjt.http.client.module.HttpMethod;
+
+import java.io.BufferedOutputStream;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Map;
+
+public class HttpRequest {
+
+    private static final String HEADER_DELIMITER = ": ";
+
+    private RequestLine requestLine;
+    private Map<String, String> requestHeader;
+    private String requestBody;
+
+    public HttpRequest (HttpMethod httpMethod, String uri, String version) {
```

Comment:
IntelliJ 의 Reformat code 기능을 활용하면 정해진 포맷으로 정렬할 수 있습니다.
```suggestion
    public HttpRequest(HttpMethod httpMethod, String uri, String version) {
```